testfiles/CmplxGlz_Daylighting_SouthVB45deg.idf

```diff
@@ -86440,7 +86440,7 @@
 
   Output:Variable,EXTWALL,Surface Outside Face Incident Sky Diffuse Solar Radiation Rate per Area,Hourly;
 
-  Output:Variable,EXTWALL,Surface Inside Temperature,Hourly;
+  Output:Variable,EXTWALL,Surface Inside Face Temperature,Hourly;
```

Comment:
OK, this is just something that got missed when the variable changed at some point.